R/CancelGeneration.R

```diff
@@ -50,23 +51,23 @@ cancelGeneration <- function(id, baseUrl, sourceKey, category) {
   checkmate::assertSubset(x = category, choices = argument$categoryStandard)
   checkmate::reportAssertions(errorMessage)
 
-  if (!all(isValidSourceKey(sourceKeys = sourceKey, baseUrl = baseUrl))) {
+  if (!all(isValidSourceKey(sourceKeys = sourceKey, baseUrl = wc$baseUrl))) {
     err <- paste0(sourceKey, " is not present in WebApi.")
     ParallelLogger::logError(err)
     stop(err)
   }
 
-  urlRoot <- paste0(baseUrl, "/", argument$categoryUrl, "/", id, "/", argument$categoryUrlCancel)
+  urlRoot <- paste0(wc$baseUrl, "/", argument$categoryUrl, "/", id, "/", argument$categoryUrlCancel)
   url <- paste0(urlRoot, "/", sourceKey)
 
   if (argument$categoryStandard %in% c("cohort")) {
-    response <- httr::GET(url)
+    response <- GET(url, authHeader = wc$authHeader)
```

Comment:
I don't believe it is a good practice to drop the namespace of the function you are calling.  `httr::GET` is appropriate here, not a blind `GET`